device/vxlan/destinationipmode.yaml

```diff
@@ -0,0 +1,16 @@
+components:
+  schemas:
+    DestinationIPMode:
```

Comment:
not required to specify,since it is IXN specific and no current use seen

Reply:
Should add "Outer IGMP Mode" as "Outer Multicast Mode" with PIM as a new option,if muticast mode is not there ,multicast forwarding might not happen in dut

Reply:
As discussed, the communication of VM information as can be done using either multicast or unicast. keeping the destionation mode as is.